### DISCUSSION TITLE
I can’t enter the submarine from the station. The hatch doesn't open.

### DISCUSSION THREAD

**Discussion by KoteNalune** [FakeFishGames/Barotrauma#14231] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

I can’t enter the submarine from the station. The hatch doesn't open. Because of this error, I can’t pick up items from the submarine for crafting and I don’t see any reason to continue the game. It's good that I didn't spend much time in the game. 

![ereerer](https://github.com/FakeFishGames/Barotrauma/assets/174135786/eafaede5-f5db-44ff-9631-915d26475a42)

### Reproduction steps

I don't know how it happened, I was just playing.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by SomeRandomNoobKekeke** [comment]
Which sub?

**Comment by Regalis11** [comment]
Thank you for the report! I think we need some additional information to diagnose the issue. Which submarine are you playing with? Did you by any chance do any changes to the wiring of the docking port or any other wiring in the sub? Would it be possible to share us the save file (note that you need to zip the file first because github doesn't allow attaching files with the .save file extension)? Sorry for the inconvenience!

**Reply by KoteNalune** [reply]
Boat Barsuk. I didn’t do anything with the wiring because I’m a newbie and don’t understand anything yet. I even made a second new game to check how the wires were installed there. At first I thought that the NPCs had messed something up.  

[2 блин.zip](https://github.com/user-attachments/files/16036550/2.zip)

**Reply by SomeRandomNoobKekeke** [reply]
Because you welded the hatch

Take your mechanic
Swim like this without a suit
![20240628234450_1](https://github.com/FakeFishGames/Barotrauma/assets/122838333/c4f4cb3b-39e5-47f7-9636-2f2b08b36b4b)
Cut that hatch with plasma cutter
Undock with no mission / dock back
Should be fixed 
![barodev small](https://github.com/FakeFishGames/Barotrauma/assets/122838333/fc00d4c6-3bcf-4a4e-b40f-69096ea5933b)

**Reply by KoteNalune** [reply]
Thank you very much for your help! I definitely couldn't do it because the bots are repairing the ship. Laughed from this incident :)

**Reply by Regalis11** [reply]
That explains the issue then, glad it was resolved. :D
